### PR TITLE
Update Puppet Providers overview

### PR DESCRIPTION
**Pull request by Soumi7**
This PR updates Puppet providers overview section in references.

References issue #890 

ScreenShots :

Screenshot from 2021-07-17 18-39-06

Screenshot from 2021-07-17 18-39-16

Screenshot from 2021-07-17 18-39-21

Screenshot from 2021-07-17 18-39-29

Screenshot from 2021-07-17 18-39-31

### CODE REVIEW

**Review by shraddhavp — APPROVED** [review]

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/puppet-providers/overview.mdx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 import Badges from './transclusions/badges.mdx'
 
 <Badges />
 
-For different instant messaging (IM) systems (such as WeChat, Whatsapp, and TikTok), the Wechaty community builds separate RPA modules for each, and we call those modules [Wechaty Puppet](https://github.com/wechaty/wechaty-puppet).
+The Wechaty community builds separate RPA modules for for different instant messaging (IM) systems (such as WeChat, Whatsapp, and TikTok). These modules are called [Wechaty Puppet](https://github.com/wechaty/wechaty-puppet).
+
+The Wechaty Puppet Provider which is supporting WeChat is named wechaty-puppet-wechat. The puppet provider supporting Whatsapp is named wechaty-puppet-whatsapp, and supporting Lark is named wechaty-puppet-lark.
 
 Wechaty Puppet Provider is an NPM module for a specific IM protocol Wechaty Puppet.
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/overview.mdx]: Wechaty Puppet Provider is an NPM module for a specific IM protocol Wechaty Puppet. Run specific Wechaty puppet using the following command:

File: docusaurus/docs/puppet-providers/overview.mdx [hunk 2]
```diff
@@ -36,6 +38,61 @@ Now, we have the following Wechaty Puppet Providers:
...
+The WECHATY_PUPPET environment will be used as the Wechaty Puppet Provider NPM name, and then you are all set.
+
+```sh
+export WECHATY_PUPPET=wechaty-puppet-whatsapp
+ts-node bot.ts
+```
+
+The above shell commands set the ```wechaty-puppet-whatsapp``` to the environment variable ```WECHATY_PUPPET``` and our Wechaty system will use it.
+
+### Import the Wechaty Puppet manually in code
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/overview.mdx]: Import -> Importing (keep the syntax for titles aligned)

**Review by proudofsimin — APPROVED** [review]
You're doing great! Just some minor advice for you. Thank you.

File: docusaurus/docs/puppet-providers/overview.mdx (hunk 1 above)
Comment (Soumi7) [reply docusaurus/docs/puppet-providers/overview.mdx]: Updated.

File: docusaurus/docs/puppet-providers/overview.mdx (hunk 2 above)
Comment (Soumi7) [reply docusaurus/docs/puppet-providers/overview.mdx]: Changed.

**Comment by Soumi7** [comment]
@proudofsimin Thanks for the suggestions, have made changes accordingly.

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Pull request merged**